<launch>
  	<node name="extract_features_node" pkg="automatic_painting" type="extract_features_node" 
       output="screen" respawn="true">
	  <remap from="points" to="/surface" />
	  <!-- range values for variables -->
	  <param name="min_pitch" value="-10" />
	  <param name="max_pitch" value="10" />
	  <param name="min_yaw" value="-10" />
	  <param name="max_yaw" value="10" />
	  <param name="resolution" value="1" />
	</node>

	<node name="point_cloud_node" pkg="automatic_painting" type="point_cloud_node" output="screen" respawn="true">
		<!-- <remap from="input" to="/camera/depth/points" /> -->
	</node>

	<node pkg="rosbag" type="record" name="features" args="record /features -o $(find automatic_painting)/bags/features" output="screen" />
</launch>
